src/pages/app-our-work-single/app-our-work-single.scss

```diff
@@ -0,0 +1,424 @@
+.our-work-single {
+  margin-top: 76px;
+
+  @include media-breakpoint-down(md) {
+    margin-top: 59px;
+  }
+
+  header {
+    position: relative;
+    display: flex;
+    justify-content: center;
+
+    img {
+      width: 100%;
+
+      &.logo {
+        height: 260px;
+        width: 260px;
+
+        @include media-breakpoint-down(md) {
+          height: 145px;
+          width: 145px;
+        }
+
+        @include media-breakpoint-down(sm) {
+          height: 200px;
+          width: 200px;
+        }
+      }
+    }
+
+    .header-content {
+      position: absolute;
+      display: flex;
+      flex-direction: column;
+      top: 80px;
+      align-items: center;
+
+      @include media-breakpoint-down(lg) {
+        top: 40px;
+      }
+
+      h1 {
+        color: #ffffff;
+        font-family: 'Muli';
+        font-size: 48px;
+        font-weight: 600;
+        letter-spacing: 1.6px;
+        line-height: 58px;
+        width: 542px;
+        text-align: center;
+
+        @include media-breakpoint-down(md) {
+          font-size: 42px;
+        }
+
+        @include media-breakpoint-down(sm) {
+          width: 99%;
+          font-size: 48px;
+        }
+      }
+
+      p {
+        color: #ffffff;
+        font-family: 'Muli';
+        font-size: 18px;
+        font-weight: 400;
+        line-height: 36px;
+        width: 311px;
+        text-align: center;
+
+        @include media-breakpoint-down(sm) {
+          margin-bottom: 23px;
+        }
+      }
+    }
+  }
+
+  section.project-content {
+    margin-top: 80px;
+
+    .images-col {
+      position: relative;
+
+      img {
+        width: 250px;
+        height: 500px;
+
+        @include media-breakpoint-down(md) {
+          width: 130px;
+          height: 260px;
+        }
+
+        @include media-breakpoint-down(sm) {
+          width: 150px;
+          height: 300px;
+          margin-left: 50px;
+        }
+
+        &:first-child {
+          position: absolute;
+          right: 175px;
+
+          @include media-breakpoint-down(md) {
+            position: initial;
+            right: initial;
+          }
+
+          @include media-breakpoint-down(sm) {
+            position: initial;
+            right: initial;
+          }
+        }
+
+        &:last-child {
+          position: absolute;
+          top: 200px;
+          right: 50px;
+
+          @include media-breakpoint-down(md) {
+            top: 100px;
+            left: 100px;
+            right: initial;
+          }
+
+          @include media-breakpoint-down(sm) {
+            top: 80px;
+            left: 80px;
+            right: initial;
+          }
+        }
+      }
+    }
+    h2 {
+      color: #333333;
```

Comment:
Let's use the global variable name for this, I believe we have it. If not, let's create it.